### PR TITLE
Removing Snapshot tester

### PR DESCRIPTION
**Pull request by mikemcd3912**
*Issue #, if available:*
N/A
*Description of changes:*
Removing the snapshot tester from Baremetal to remedy hardcoded storage class issue

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by elamaran11 — COMMENTED** [review]
@mikemcd3912 `test-app` can still remain, we want to see if the CSI driver is creating a PVC. You can remove snapshot, snapshot-restore which is not supported in the CSI driver

**Comment by mikemcd3912** [comment]
GH action failed because we're adding/editing a file to a directory w/o a namespace associated with it, which is expected behavior

File: eks-anywhere-baremetal/Testers/Core/snapshot-tester/test-app/kustomization.yaml [hunk 1]
```diff
@@ -1,5 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-resources:
-  - claim.yaml
```
Comment (elamaran11) [inline eks-anywhere-baremetal/Testers/Core/snapshot-tester/test-app/kustomization.yaml:4]: Why is this file removed.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@mikemcd3912 One comment.

**Review by elamaran11 — APPROVED** [review]
LGTM

File: eks-anywhere-baremetal/Testers/Core/snapshot-tester/test-app/kustomization.yaml (hunk 1 above)
Comment (mikemcd3912) [reply eks-anywhere-baremetal/Testers/Core/snapshot-tester/test-app/kustomization.yaml]: Adding these two kustomization files back since we're using the v1beta1 API version that needs these elsewhere in the repository

Comment (mikemcd3912) [reply eks-anywhere-baremetal/Testers/Core/snapshot-tester/test-app/kustomization.yaml:4]: Per our discussion these resource pointer files are unnecessary
